src/main/java/com/team701/buddymatcher/domain/timetable/Course.java

```diff
@@ -32,63 +32,70 @@
     @ManyToMany(fetch = FetchType.EAGER, cascade = CascadeType.PERSIST)
     @JoinTable(
             name = "COURSE_STUDENT",
-            joinColumns = @JoinColumn(name = "id"),
-            inverseJoinColumns = @JoinColumn(name = "COURSE_ID"))
+            joinColumns = @JoinColumn(name = "COURSE_ID"),
+            inverseJoinColumns = @JoinColumn(name = "USER_ID"))
```

Comment:
@ff-dev-45 Fixed after writing integration tests

Reply:
Thanks for picking up on this and fixing it